wasm/utils/solana-extra/Cargo.toml

```diff
@@ -14,29 +14,20 @@ readme = "README.md"
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-Inflector = "0.11"
 arrayref = "0.3"
 assert_matches = "1"
 base64 = "0.13"
 bincode = "1"
 borsh = "0.10"
 bs58 = "0.4"
 bytemuck = "1"
-chrono = { version = "0.4", features = ["serde"] }
-clap = "3"
-console = "0.15"
 fluvio-wasm-timer = "0.2"
-humantime = "2"
-lazy_static = "1"
-log = "0.4.17" # only for nightly builds
```

Comment:
This was added to fix a `nightly` issue in https://github.com/solana-playground/solana-playground/pull/57. It probably doesn't exist anymore since you also removed the `solana-frozen-abi` dependency, but could you double check to make sure?

Reply:
You're right I will revert this one and work on it when we will update all the dependencies. Because currently I'm unable to test it

Reply:
- name: Codecov